nixos/modules/services/networking/shout.nix

```diff
@@ -6,6 +6,21 @@ let
   cfg = config.services.shout;
   shoutHome = "/var/lib/shout";
 
+  defaultConfig = pkgs.runCommand "config.js" {} ''
+    EDITOR=true ${pkgs.shout}/bin/shout config --home $PWD
+    mv config.js $out
+  '';
+
+  configFile = if cfg.configFile then cfg.configFile else ''
```

Comment:
Oops nice catch @joachifm, I updated the expression to put it in `configFile.default`.

Also updated the option description and added an example for `config`.